source/slides/11_chef.rst

```diff
@@ -0,0 +1,596 @@
+.. _11_chef:
+
+===========
+Chef Basics
+===========
+
+Terminology
+===========
+
+*Client*
+    A client is identified by a key pair, used for authorization.
+
+*Node*
+    A machine you run ``chef-client`` on.
+
+*Knife*
+    A tool for interacting with the chef server.
+
+*Berkshelf*
+    A tool that manages cookbooks developed by others.
+
+
+Terminology (cont)
+==================
+
+
+*Foodcritic*
+    A linting tool for chef.
+
+*Rubocop*
+    A popular ruby linting tool, similar to pep8.
+
+Even More Terminology
+=====================
+
+*Test Kitchen*
+    A framework for automatically bringing up machines, running chef, and then running tests.
```

Comment:
I avoid it because it is a tool for running integration tests more than an integration testing tool itself. You can run just unit tests with test kitchen too